includes/Core/Modules/Modules.php

```diff
@@ -177,9 +176,7 @@ public function __construct(
 		$this->authentication   = $authentication ?: new Authentication( $this->context, $this->options, $this->user_options );
 		$this->assets           = $assets ?: new Assets( $this->context );
 
-		if ( Feature_Flags::enabled( 'adsModule' ) ) {
-			$this->core_modules[ Ads::MODULE_SLUG ] = Ads::class;
-		}
```

Comment:
Let's add the Ads module to the core modules directly, similarly how we do for other modules: https://github.com/google/site-kit-wp/blob/ee48119f96675c792cb7a556edcdccad7a1f435b/includes/Core/Modules/Modules.php#L146-L153